src/tests/apps.cpp

```diff
@@ -141,6 +141,44 @@ void translationCache(AsyncService &service, Ptr<TranslationModel> model) {
   std::cout << firstResponse.target.text;
 }
 
+void tagTranslationBlockingService(Ptr<TranslationModel> model) {
```

Comment:
I'd prefer to put this with `AsyncService` to be consistent with the rest. If you're having trouble, I can help if permitted.